include/seqan3/alignment/pairwise/edit_distance_unbanded.hpp

```diff
@@ -1054,11 +1057,10 @@ class edit_distance_unbanded :
         if constexpr (compute_end_positions)
             cached_end_positions = this->end_positions();
 
-        if constexpr (compute_begin_positions)
+        if constexpr (compute_begin_positions && !compute_sequence_alignment)
```

Comment:
@rrahn You used `traits_type::` further down in this function, why did you use `compute_begin_positions` here?

Should this be

```suggestion
        if constexpr (traits_type::compute_begin_positions && !traits_type::compute_sequence_alignment)
```
?

Reply:
Nop, there is a dependency within the algorithm independent of what should  be presented in the alignment result. Before, we said that if I want to output the alignment I also have to compute the end and begin positions. That is resolved in the edit distance via specific policies that are enabled based on the properties set by the edit distance traits class. Now, we changed the behavior to say that I only produce what is requested by the user. But still, the edit distance requires to enable the begin position policy in order to compute the alignment. So in the edit traits class we enable everything based on the old behavior, i.e. in order to compute the alignment we also have to compute the begin position but in the alignment result we only store the alignment and not the begin position. So compute_begin_position is different to traits_type::compute_begin_position, by that the former one is used and set internally by the algorithm and the latter one reflects the request of the user.